### DISCUSSION TITLE
Any options/docs for Mobile Development?

### DISCUSSION THREAD

**Discussion by Blankeos** [maxnowack/signaldb#763] [General]
Signal DB looks promising! I was wondering if it was compatible with any of these mobile technologies?

- React Native
- Capacitor
- Flutter

I'm looking for an alternative to MongoDB Realm and Watermelon DB.

**Comment by maxnowack** [comment]
SignalDB should be compatible every mobile technology that uses a javascript environment. So it should work with the technologies you mentioned 🙂
At the moment there is no clear tutorial how to use SignalDB with these technologies, but it shouldn't differ much from the general usage.
The only thing you maybe want to have in addition is a persistence adapter that you can persist your data on the device. What exactly you need depends a bit on your use case and the platform you're using. Take a look at the [documentation how to implement a custom persistence adapter](https://signaldb.js.org/data-persistence/other/). I'm happy to help 🙂
